app/views/article.html

```diff
@@ -20,7 +20,7 @@
     <meta name="msapplication-TileColor" content="#37dc8c">
     <meta
       name="google-signin-client_id"
-      content="520465458218-e9vp957krfk2r0i4ejeh6aklqm7c25p4.apps.googleusercontent.com"
```

Comment:
This GSI Client ID appears in several other templates. Could this PR update them as well?

Reply:
It also appears in two JS files

Reply:
Doing so for the html files will be straight-forward, but the js files... I'm thinking I'd create a route to render them in a similar way to the html files. That'd be easy, but I'd have to look at the build/release process after to see if the files are used elsewhere like to create a concatenated bundle, which would then fail. I'll investigate.

Reply:
Ok, this is the wrong approach. Think I have to do it in the gulp build.

Reply:
I was able to do the js replacement in the `gulp` process.